processor/batchprocessor/batch_processor_test.go

```diff
@@ -124,114 +124,100 @@ func TestBatchProcessorSpansDeliveredEnforceBatchSize(t *testing.T) {
 
 func TestBatchProcessorSentBySize(t *testing.T) {
 	sizer := &ptrace.ProtoMarshaler{}
-	views := MetricViews()
-	require.NoError(t, view.Register(views...))
-	defer view.Unregister(views...)
-
-	sink := new(consumertest.TracesSink)
-	cfg := createDefaultConfig().(*Config)
-	sendBatchSize := 20
-	cfg.SendBatchSize = uint32(sendBatchSize)
-	cfg.Timeout = 500 * time.Millisecond
-	creationSet := componenttest.NewNopProcessorCreateSettings()
-	batcher, err := newBatchTracesProcessor(creationSet, sink, cfg, configtelemetry.LevelDetailed)
-	require.NoError(t, err)
-	require.NoError(t, batcher.Start(context.Background(), componenttest.NewNopHost()))
+	telemetryTest(t, func(t *testing.T, tel testTelemetry, registry *featuregate.Registry) {
```

Comment:
The changes in this file would probably be easier to ready if you introduced a new top-level function everywhere `telemetryTest` is used, to preserve indentation.

```
func testBatchProcessorSentBySize(t *testing.T, tel testTelemetry, registry *featuregate.Registry) {
   ... mostly same code not re-indented
}
```

Reply:
Thanks for the suggestion, hopefully it more readable now :)